CHANGELOG.md

```diff
@@ -0,0 +1,86 @@
+# Changelog
+
+Releases are now named after famous [Cricket](https://en.wikipedia.org/wiki/Cricket) players.
+
+## v0.4 "Akram": Stability, HTTPS & breaking changes
+
+Released 2017-06-23
+
+This version has a bunch of breaking changes. If you are upgrading from an older 
+version that is using any of the features that changed (you probably are!) and 
+do not know much about Kubernetes, we recommend just deleting and recreating your
+installation!
+
+If you need support, reach out to us on
+[gitter](https://gitter.im/jupyterhub/jupyterhub) or open an
+[issue](https://github.com/jupyterhub/helm-chart/issues).
+
+### Breaking changes ###
+
+* Names of user pods and dynamically created home directory PVCs no longer include
+  the userid in them by default. If you are using dynamic PVCs for home directories
+  (which is the default), you will need to manually rename them before upgrading.
+  Otherwise, new PVCs will be created, and users might freak out about their home
+  directories appearing empty. 
+  
+  See [this pull request](https://github.com/jupyterhub/kubespawner/pull/56) on
+  what needs to change! 
+
+* A [StorageClass](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#storageclasses)
+  is no longer created by default. This shouldn't affect most new installs,
+  since as of Kubernetes 1.6 most cloud provider installations have a default.
+  If you are using an older version of kubernetes, easiest thing to do is upgrade
+  to a newer version. If not, you can create a StorageClass manually and everything
+  should continue to work.
+  
+* Setting `token.proxy` no longer works - set `proxy.secretToken` instead.
+  If your `config.yaml` contains something that looks like the following:
+  
+  ```yaml
+  token:
+      proxy: <some-secret>
+  ```
+  
+  you can / should change that to:
+  
+  ```yaml
+  proxy:
+      secretToken: <some-secret>
+  ```
+
+
+### Features ###
+
+* GitHub Authentication is now supported, thanks to [Jason Kuruzovich](https://github.com/jkuruzovich).
+* Using an [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) is
+  now supported! If your cluster already has Ingress support (with automatic Let's Encrypt support, perhaps),
+  you can easily use that now.
+* We now add a label to user pods / PVCs with their usernames.
+* Support using a static PVC for user homes or for the hub db. This makes this release usable
```

Comment:
user `home` directories or the hub database... where ~~you~~ only ~~have~~ one NFS share is used for the whole hub.